docs/proposals/kubernetes-mcpgroup-crd.md

```diff
@@ -0,0 +1,272 @@
+# MCPGroup CRD for Kubernetes Operator
```

Comment:
In the CLI, we create a default group and all servers must belong to a group. This is different in k8s where servers don't need to belong to a group. I think this makes sense, but I wanted to note it so we're aware of the differences.